bedrock/mozorg/templates/mozorg/home/includes/pocket-best-of.html

```diff
@@ -0,0 +1,28 @@
+{#
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+{% if LANG == 'de' %}
+  {% set small_text = 'Vorgestelltes Produkt' %}
+  {% set heading_text = 'Entdecke die besten Artikel aus 2023' %}
+  {% set cta_text = 'Zum Best of 2023' %}
+{% else %}
+    {% set small_text = 'Featured product' %}
+    {% set heading_text = 'Discover the top articles of 2023' %}
+    {% set cta_text = 'See the winners' %}
+{% endif %}
+
+<aside class="pocket-feature mzp-l-content">
+  <div class="feature-wrapper">
+    <div class="image-wrapper">
+      <img loading="lazy" src="{{ static('img/home/2023/pocket-best-of.png') }}" alt="Pocket: Best of 2023">
```

Comment:
Question: I assume this is intentional, but should it still be "Best of 2023" even for German?

Reply:
Yeah, I brought this up, but German copywriter suggested that this image was ok since Best translates to Best in german